apps/condo/domains/common/components/containers/FormList.tsx

```diff
@@ -252,30 +252,7 @@ const FormWithAction: FunctionComponent<IFormWithAction> = (props) => {
             return handleSubmit(values)
         }
         if (values.hasOwnProperty(NON_FIELD_ERROR_NAME)) delete values[NON_FIELD_ERROR_NAME]
-        let data
```

Comment:
We have 9 usages of FormWithAction in our code
Did you check that all of them now handle errors correctly?

Reply:
as for me it's better to leave it here %)

Reply:
ok, i return back this logic and now i will test all FormWithAction

Reply:
Checked crud on all forms where used `FormWithAction`, works fine